httplib2_transport/pylintrc

```diff
@@ -0,0 +1,165 @@
+[MASTER]
```

Comment:
Contents of this file worth looking at? Also, why don't you just refer to the RC for the main project?